### PR TITLE
(bugsbash) Error string should not be captilazed in options.go

### PR DESCRIPTION
**Pull request by asquare14**
Signed-off-by: aSquare14 <atibhi.a@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes
Fixes ST1005 : Error string should not be capitalized and should not end with punctuation or a newline

### CODE REVIEW

**Review by squat — DISMISSED** [review]
this one is tricky because this error is not wrapped any more but is the result of parsing command line arguments.

If other maintainers prefer this output be a full sentence, then we can refrain from merging

File: pkg/logging/options.go
Comment (onprem) [inline pkg/logging/options.go:178]:
```suggestion
		return logOpts, fmt.Errorf("both log.request.decision and request.logging has been enabled, please use only one of the flags")
```

As @squat said this is a bit tricky, but I'll still prefer no-caps to be consistent. After all this is going to be printed in a structured format `err="both log......"` so it's better to be consistent.

**Review by onprem — APPROVED** [review]
Just a small nit. Everything else LGTM.

**Comment by asquare14** [comment]
@onprem and @squat Thanks for the review. Fixed the suggestions :)

File: pkg/logging/options.go
Comment (squat) [inline pkg/logging/options.go:204]: let's make this one match above:

```suggestion
		return []tags.Option{}, logOpts, fmt.Errorf("both log.request.decision and request.logging-config have been enabled, please use only one of the flags")
```

File: pkg/logging/options.go
Comment (squat) [inline pkg/logging/options.go:178]:
```suggestion
		return logOpts, fmt.Errorf("both log.request.decision and request.logging have been enabled, please use only one of the flags")
```

**Review by squat — COMMENTED** [review]
one more iteration plz! let's make sure they are consistent with each other

**Review by squat — DISMISSED** [review]
:shipit:

**Comment by squat** [comment]
ah, need to sign-off the DCO one more time it seems

**Comment by asquare14** [comment]
> ah, need to sign-off the DCO one more time it seems

Done 😛 
Thanks for the review 🚀

**Review by squat — APPROVED** [review]

**Pull request merged**
